features/importing/work_import_dw.feature

```diff
@@ -43,35 +42,30 @@ Feature: Import Works from DW
   # DW metadata tables esp. when there's no DW metadata table
 
     Given basic tags
-      And a fandom exists with name: "Lewis", canonical: true
       And the following activated user exists
         | login          | password    |
         | cosomeone      | something   |
       And I am logged in as "cosomeone" with password "something"
     When I go to the import page
-      And I fill in "urls" with "http://rebecca2525.dreamwidth.org/3601.html"
+      And I fill in "urls" with "https://ao3testing.dreamwidth.org/1836.html"
     When I press "Import"
     Then I should see "Preview"
-      And I should see "Lewis" within "dd.fandom"
-      And I should see "General Audiences" within "dd.rating"
-      And I should see "Lewis/Hathaway" within "dd.relationship"
-      And I should see "Published:2000-01-10"
-      And I should see "Importing Test" within "h2.title"
-      And I should not see "[FIC]" within "h2.title"
-      And I should see "Something I made for testing purposes." within "div.summary"
-      And I should see "Yes, this is really only for testing. :)" within "div.notes"
-      And I should see "My first paragraph."
-      And I should see "My second paragraph."
+      And I should see "Testing" within "dd.fandom"
+      And I should see "Teen And Up Audiences" within "dd.rating"
+      And I should see "Character A/Character B" within "dd.relationship"
+      And I should see "Published:2017-07-03"
+      And I should see "Single Chapter Fic from DW" within "h2.title"
+      And I should not see "NFS" within "h2.title"
```

Comment:
This might be confusing for future folks -- "NFS" isn't in the post in its present form, but I think it was in the post before you edited it?